example/simple-request-anonymized.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Scenario:
+ * Generate requests towards specific domain
+ **/
+var trafficSimulator = require('../lib/main');
+var anonymizer = require('../lib/anonymizer');
```

Comment:
The anonymizer dependency should not be imported here. You anonymizer feature should be exposed through the main library